prow/gcsupload/run.go

```diff
@@ -19,6 +19,8 @@ package gcsupload
 import (
 	"context"
 	"fmt"
+	"io"
+	"io/ioutil"
```

Comment:
We recently landed a [PR](https://github.com/kubernetes/test-infra/pull/27674) that migrates [deprecated](https://pkg.go.dev/io/ioutil) `io/ioutil` usage, can you please adapt the PR for that?

Reply:
The PR uses `io/ioutil` no more, thanks for the advice.